docs/slack.rst

```diff
@@ -27,16 +27,24 @@ Feel free to send a Pull Request to update `this page`_, if you want to include
 Slack History
 -------------
 
-We don't pay for our Slack, which means that the community has access to only the most
-recent 10,000 messages. Paying for Slack is prohibitively expensive for a community
-like ours, but many people seem to feel a freedom in knowing that their words will disappear shortly.
+We don't pay for our Slack, which means that the community has access to only the most recent 10,000 messages.
+Paying for Slack is prohibitively expensive for a community like ours,
+but many people seem to feel a freedom in knowing that their words will disappear shortly.
 We understand that not everyone loves this decision,
-but it's the way things are currently set up and we don't plan to change it in the near future. 
+but it's the way things are currently set up
+and we don't plan to change it in the near future.
```

Comment:
Maybe we could also change "near future" to "foreseeable future" to make it a little clearer that fetch isn't going to happen?

Reply:
Good idea! I tightened it up even more. "We understand that not everyone loves this decision,
but we don't plan to change it in the foreseeable future."

Reply:
Love it!

Reply:
We might even just drop the timing entirely.